wix-members-frontend/wix-members-frontend/Authentication.service.json

```diff
@@ -72,12 +73,13 @@
                 { "title": "Register a member using a 3rd party for approval",
                   "body":
                     [ "/*******************************",
-                      " * Backend code - register.jsw *",
+                      " * Backend code - register.web.js *",
                       " *******************************/",
+                      "import { Permissions, webMethod } from 'wix-web-module';",
                       "import { authentication } from 'wix-members-backend';",
                       "import { approveBy3rdParty } from 'backend/some-backend-module';",
                       "",
-                      "export async function doRegistration(email, password, firstName, lastName) {",
+                      "export const doRegistration = webMethod(Permissions.Anyone, async (email, password, firstName, lastName) => {",
```

Comment:
Code examples weren't part of your original developer-docs PR. This is part of the web.js migration, so you're probably good to merge, but make sure you sync with Adam because I think he's keeping track of what code examples are being merged.